python/lsst/sims/catUtils/mixins/PhotometryMixin.py

```diff
@@ -148,6 +148,22 @@ def get_lsst_photometric_uncertainties(self):
                                                  'lsstBandpassDict')
 
 
+    def calculateVisibility(self, magFilter, m5, sigma=0.12):
+        """
+        Calculate the probability of detecting a particular source.
```

Comment:
This doesn't actually return the probability of an object being visible.  It returns a truth value (i.e. was it seen or wasn't it).

Should this method be split into two (one that calculates the probability that an object is visible; one that determines whether or not the object was seen) so that users have the option of outputting catalogs with a probability column?